package.json

```diff
@@ -51,7 +53,10 @@
     "@biomejs/biome": "^1.9.4",
     "@types/node": "^22.15.21",
     "@types/picomatch": "^4.0.0",
+    "fast-glob": "^3.3.3",
     "fs-fixture": "^2.7.1",
+    "glob": "^11.0.2",
```

Comment:
Adding `globby` as well might be nice

Reply:
do we really need to? it's pretty much a fast-glob wrapper making it unlikely to ever show faster times than fast-glob

Reply:
it might be interesting to do once to see if there is a difference or not. I thought there could be a possibility that some of its extra functionality like `expandDirectories` could slow things down